eng/bomgenerator/inputDependencies.txt

```diff
@@ -0,0 +1,24 @@
+com.azure:azure-core:1.17.0;
+com.azure:azure-core-http-netty:1.10.0;
```

Comment:
I wouldn't do that because it can trip the current tool and here is why. We try to consolidate all the versions of a given group_artifact combination and then from the list of versions in the closure, choose 1 to base all the libraries against. Allowing another version will result in changing that logic to also include this version which complicates things more than necessary.